SmartSearch.LuceneNet/Internals/SpecializedFields/HigherRelevanceLiteralField.cs

```diff
@@ -0,0 +1,66 @@
+﻿using SmartSearch.Abstractions;
+using SmartSearch.LuceneNet.Analysis;
+using System;
+using System.Globalization;
+using System.Text;
+
+namespace SmartSearch.LuceneNet.Internals.SpecializedFields
+{
+    internal class HigherRelevanceLiteralField : Field, ISpecializedField
```

Comment:
This is the special literal field that is created for text fields with Relevance.Higher on them.